### ISSUE TITLE
visdom 0.1.8.2 very slow

### ISSUE THREAD

**Issue by jabacrack** [facebookresearch/visdom#356]
Windows 10, conda,  python 3.6.4, visdom 0.1.8.2
run code 
```
import visdom
import numpy as np

vis = visdom.Visdom()

for i in range(10):
    data = np.random.randn(17, 110)
    vis.line(data)
```

Each plot draw near 5 seconds(visdom console print "INFO:tornado.access:200 POST /events (::1) 6306.59ms"), in previous version that I use (0.1.7) all plots created in 2-3 seconds.

**Comment by JackUrb** [comment]
This is a bug introduced in 1.8.2 that's being addressed in #355

**Comment by JackUrb** [comment]
#355 is in
